src/Components/Listing.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styles from './styles.css';
+
+class Listing extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {}
+  }
+
+  render() {
+    const { data } = this.props;

```

Comment:
For clarification, this will be a reference to the props object, not what you've passed down through props. Is that what you want it to be?

Reply:
@harrisonywu no its deconstruction I am pretty sure eslint wanted it this way so that's why I did it